cli/ops.rs

```diff
@@ -341,6 +341,12 @@ fn op_start(
 
   let main_module = state.main_module().map(|m| builder.create_string(&m));
 
+  let xeval_delim = state
+    .flags
+    .xeval_delim
+    .clone()
```

Comment:
Is the clone necessary?

Reply:
I believe yes, we are doing a move later as we initialize the `StartResArgs` struct (while `state.flags.log_debug` copies by default)